include/libc/math.h

```diff
@@ -8,6 +8,7 @@
 #define MAXFLOAT 3.40282347e+38f
 #define SHT_MAX 32767.0f
 #define SHT_MINV (1.0f / SHT_MAX)
+#define SQ(x) ((x)*(x))
```

Comment:
Since this is a standard C header, it should really only contain standard C macros. libultra probably should be changed so it doesn't use this and put it in z64math.h instead or something